proxy/src/http/control.rs

```diff
@@ -115,23 +126,26 @@ mod handler {
 
     /// Create a project from the fixture repo.
     pub async fn create_project(
-        librad_paths: Arc<RwLock<Paths>>,
+        peer: Arc<Mutex<coco::Peer>>,
+        owner: http::Shared<coco::User>,
         input: super::CreateInput,
     ) -> Result<impl Reply, Rejection> {
-        let dir = tempfile::tempdir().expect("tmp dir creation failed");
-        let paths = librad_paths.read().await;
-        let (id, meta) = coco::replicate_platinum(
-            &dir,
-            &paths,
-            &input.name,
-            &input.description,
-            &input.default_branch,
-        )?;
+        let owner = &*owner.read().await;
+        let mut peer = peer.lock().await;
+
+        let meta = peer
+            .replicate_platinum(
```

Comment:
Ah `replicate_platinum` is here in pub code. I think we should create a ticket to remove this since we're able to browse existing projects now and whatnot. So creating a project should really be creating a project from scratch.